src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -551,6 +551,7 @@ public static void IsNormal(double d, bool expected)
         }
 
         [Theory]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/38455", TestRuntimes.CoreCLR)]
```

Comment:
I'm not sure if it's preferable to use that, or something like:
```
[SkipOnCoreClr("https://github.com/dotnet/runtime/issues/38455")]
```

Reply:
Whenever there is an issue open to fix that we try to use `ActiveIssue`... whenever there is a reason that we don't plan to fix it because it is a long running test or it is a test stress mode that we want to disable on, we use `SkipOnCoreClr`.